torch/optim/swa_utils.py

```diff
@@ -162,30 +167,33 @@ class AveragedModel(Module):
     .. _Polyak averaging:
         https://paperswithcode.com/method/polyak-averaging
     """
-    def __init__(self, model, device=None, avg_fn=None, multi_avg_fn=None, use_buffers=False):
+
+    def __init__(
+        self,
+        model: Module,
+        device: Optional[Union[int, device]] = None,
+        avg_fn: Optional[Callable[[Tensor, Tensor, Tensor], Tensor]] = None,
+        multi_avg_fn: Optional[Callable[[Tensor, Tensor, Tensor], Tensor]] = None,
```

Comment:
These look wrong

Reply:
Fixed.
I also see some lint error from the CI, should I fix them?